player.js

```diff
@@ -0,0 +1,41 @@
+
+var Scrabble = require("./scrabble.js");
+var scrabble = new Scrabble
+
+function Player(name) {
+  this.name = name;
+  this.plays = [];
+};
+
+Player.prototype.play = function(word) {
+  this.plays.push(word);
+};
+
+Player.prototype.total_score = function() {
+  var total = 0;
+  for (var word of this.plays) {
+    total += scrabble.score(word);
+  };
+  return total;
+};
+
+Player.prototype.hasWon = function() {
+  var total = Player.total_score;
+  if (total >= 100) {
```

Comment:
just realized- you could one better than ternary here! you can just do: `return (total >= 100)`, which will be a boolean :D